metadata-service/auth-filter/src/test/java/com/datahub/auth/authentication/AuthenticationFilterTest.java

```diff
@@ -31,33 +33,35 @@ public class AuthenticationFilterTest extends AbstractTestNGSpringContextTests {
 
   @Autowired AuthenticationFilter authenticationFilter;
 
-  @Autowired StatefulTokenService statefulTokenService;
-
   @Test
-  public void testExpiredToken() throws ServletException, IOException, TokenException {
+  public void testExpiredToken() throws ServletException, IOException, AuthenticationException {
     FilterConfig mockFilterConfig = mock(FilterConfig.class);
     when(mockFilterConfig.getInitParameterNames()).thenReturn(Collections.emptyEnumeration());
 
     authenticationFilter.init(mockFilterConfig);
     HttpServletRequest servletRequest = mock(HttpServletRequest.class);
     HttpServletResponse servletResponse = mock(HttpServletResponse.class);
     FilterChain filterChain = mock(FilterChain.class);
-    Actor actor = new Actor(ActorType.USER, "datahub");
-    //    String token = _statefulTokenService.generateAccessToken(TokenType.SESSION, actor, 0L,
-    // System.currentTimeMillis(), "token",
-    //        "token", actor.toUrnStr());
-    // Token generated 9/11/23, invalid for all future dates
-    String token =
-        "eyJhbGciOiJIUzI1NiJ9.eyJhY3RvclR5cGUiOiJVU0VSIZCI6ImRhdGFodWIiLCJ0eXBlIjoiU0VTU0lPTiIsInZlcnNpb24iOiIxIiwian"
-            + "RpIjoiMmI0MzZkZDAtYjEwOS00N2UwLWJmYTEtMzM2ZmU4MTU4MDE1Iiwic3ViIjoiZGF0YWh1YiIsImV4cCI6MTY5NDU0NzA2OCwiaXNzIjoiZGF"
-            + "0YWh1Yi1tZXRhZGF0YS1zZXJ2aWNlIn0.giqx7J5a9mxuubG6rXdAMoaGlcII-fqY-W82Wm7OlLI";
+
+    // Setup for expired token test
     when(servletRequest.getHeaderNames())
         .thenReturn(Collections.enumeration(List.of(AUTHORIZATION_HEADER_NAME)));
-    when(servletRequest.getHeader(AUTHORIZATION_HEADER_NAME)).thenReturn("Bearer " + token);
```

Comment:
This no longer tests the expected behavior? The token generated is specifically one that has an expiration date, this no longer verifies that we are verifying the expiresAt field.

Reply:
This test fails for the wrong reason, based on the name of the test testExpiredToken included the expectation that the test fails. However the reason it was failing was not due to the token expiration, but an invalid token signature. Updated the test to test the expiration exception handling.

Reply:
Hmm, are you sure? When I created this it was definitely verifying the expiresAt part of the token. I specifically generated a valid token that would fail after the commit got actually merged in since it was only valid for 24 hours. If this is the case would prefer that the token be regenerated to have the correct header rather than putting in a obviously invalid "expired-token" token.

Reply:
Yes, it was being returned as invalid, but not due to the expiration, it was a failed signature check.